includes/controls/groups/background.php

```diff
@@ -814,4 +827,20 @@ protected function get_default_options() {
 			'popover' => false,
 		];
 	}
+
+	protected function get_images_sizes() {
+		$sizes = get_intermediate_image_sizes();
+		$sizes[] = 'full';
+		$result = [];
+		foreach ( $sizes as $size ) {
+			$width = get_option( "{$size}_size_w" );
+			$height = get_option( "{$size}_size_h" );
+			if ( ! $width && ! $height ) {
+				$result[ $size ] = $size;
+			} else {
+				$result[ $size ] = $size . ' (' . $width . 'x' . $height . ')';
```

Comment:
The option labels need to be translated

Reply:
The options on the images sizes are not translated also.

Reply:
Is the `$size` string translated?

Reply:
Yes